source/php/Module/views/card-template.blade.php

```diff
@@ -2,9 +2,10 @@
       @if($recommendLinkList)
             @foreach($recommendLinkList as $recommendLink)
                 @include('partials.card', [
-                    "text" => $recommendLink->recommendLinkLabel,
-                    "href" => $recommendLink->recommendLinkTarget,
-                    "type" => "static",
+                    "heading" => $recommendLink->recommendLinkLabel,
+                    "content" => get_the_excerpt($recommendLink->recommendLinkTarget),
```

Comment:
Functions in views should be avoided. Functionality should be placed in controller.